test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -1161,4 +1199,164 @@ describe('global navigation', () => {
       expect(getOverflowingTopnav()).to.equal(null);
     });
   });
+
+  describe('Universal navigation', () => {
+    let unavMeta;
+    const orgAlloy = window.alloy;
+    beforeEach(async () => {
+      window.UniversalNav = sinon.spy();
+      // eslint-disable-next-line no-underscore-dangle
+      window._satellite = { track: sinon.spy() };
+      // eslint-disable-next-line
+      window.alloy = () => new Promise((resolve) => resolve({ identity: { ECID: 'dummy-ECID' } }));
+      unavMeta = toFragment`<meta name="universal-nav" content="on">`;
+      document.head.append(unavMeta);
+    });
+
+    afterEach(() => {
+      sinon.restore();
+      window.allow = orgAlloy;
+      if (document.head.contains(unavMeta)) document.head.removeChild(unavMeta);
+    });
+
+    describe('desktop', () => {
+      it('should render the Universal navigation', async () => {
+        const fullUnavMeta = toFragment`<meta name="universal-nav" content="profile, appswitcher, notifications, help">`;
+        const gnav = await createFullGlobalNavigation();
+        await gnav.decorateUniversalNav();
+        const unavFirstCallItems = window.UniversalNav.getCall(0).args[0]?.children;
+
+        expect(unavFirstCallItems[0]?.name === 'profile' && !unavFirstCallItems[1]).to.be.true;
+
+        document.head.removeChild(unavMeta);
+        document.head.append(fullUnavMeta);
+        await gnav.decorateUniversalNav();
+        const unavSecondCallItems = window.UniversalNav.getCall(1).args[0]?.children;
+
+        expect(unavSecondCallItems.every((c) => ['profile', 'app-switcher', 'notifications', 'help'].includes(c.name)))
```

Comment:
Should we rather verify, the elements have been added to the DOM?

Reply:
The notifications are not added unless you are logged-in, so everything depends on the Universal Navigation. I don't think we need to check for the elements, just that the correct children have been passed.

Reply:
Yes, that would fall under the responsibility of the Universal Navigation, we just need to make sure we are sending the correct children